jina/peapods/container.py

```diff
@@ -65,37 +67,50 @@ def post_init(self):
         _args = kwargs2list(non_defaults)
         self._container = self._client.containers.run(self.args.uses, _args,
                                                       detach=True, auto_remove=True,
-                                                      ports={'%d/tcp' % v: v for v in
+                                                      ports={f'{v}': v for v in
                                                              _expose_port},
                                                       name=self.name,
                                                       volumes=_volumes,
                                                       network_mode=net_mode,
                                                       entrypoint=self.args.entrypoint,
-                                                      # network='mynetwork',
-                                                      # publish_all_ports=True
+                                                      # publish_all_ports=True # This looks like something I would activate
                                                       )
         # wait until the container is ready
         self.logger.info('waiting ready signal from the container')
 
     def loop_body(self):
         """Direct the log from the container to local console """
-        import docker
+        import asyncio
+        def check_ready():
+            import asyncio
+            while not self.is_ready:
+                asyncio.sleep(0.1)
+            self.is_ready_event.set()
+            self.logger.success(__ready_msg__)
+            return True
+
+        async def _loop_body():
+            import asyncio
```

Comment:
Can we move the import to the top of the file, since this is a standard Python import?

Reply:
Done!